docs/user/settings.rst

```diff
@@ -742,3 +742,54 @@ recoverable failures, improving the reliability of the system.
 For more information on these settings, you can refer to the `the celery
 documentation regarding automatic retries for known errors.
 <https://docs.celeryq.dev/en/stable/userguide/tasks.html#automatic-retry-for-known-exceptions>`_
+
+``OPENWISP_CONTROLLER_WHO_IS_ENABLED``
+--------------------------------------
+
+============ =========
+**type**:    ``bool``
+**default**: ``False``
+============ =========
+
+Allows to enable WhoIs lookup feature.
+
+.. image:: https://github.com/user-attachments/assets/0737d39c-1fad-4fca-aca9-9b42bc321763
+    :alt: who_is admin setting
+
+This feature is used to fetch details of a device based on its last
+reported public IP address. The fetched details include ASN, CIDR,
+address, timezone and name of organization associated with ASN.
```

Comment:
Make sure this is explained in the dedicated whois page, we can remove this from here to avoid redundancy and make the text more maintainable.